docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -129,21 +136,31 @@ researchers at Amgen, Inc, and you can read the paper in which it was published
 
 .. image:: MM_graph_time_varying_rates.jpg
 
-.. _allowing_other_complexities:
 
-Allowing other complexities
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Surfacing limitations
+~~~~~~~~~~~~~~~~~~~~~
 
 Another benefit of microsimulations in general is that they can surface complications or limitations that are obscured in non-individual-based models, such as compartmental models.
```

Comment:
Maybe actually instead of cohort analysis we can just use non-individual-based models?